reg_tests/rt.sh

```diff
@@ -88,8 +88,8 @@ cd ../reg_tests
 
 set -x
 
-#if [[ $target == "orion" ]] || [[ $target == "jet" ]] || [[ $target == "hera" ]] || [[ $target == "hercules" ]] || [[ $target == "wcoss2" ]] ; then
-#if [[ $target == "orion" ]] || [[ $target == "jet" ]] || [[ $target == "hera" ]] || [[ $target == "hercules" ]] ; then
+#if [[ $target == "orion" ]] || [[ $target == "jet" ]] || [[ $target == "hercules" ]] || [[ $target == "wcoss2" ]] ; then
+#if [[ $target == "orion" ]] || [[ $target == "jet" ]] || [[ $target == "hercules" ]] ; then
```

Comment:
Do we need to keep if they're going to stay commented out?

Reply:
Cleaned up script at ecfced3. FYI. This is the script that is started off the cron on each machine (using the role account)